### PR TITLE
[CS ] czech - add responses

### PR DESCRIPTION
**Pull request by schizza**
default responses added.
Cover changes to sentences files as required due to #691

### CODE REVIEW

File: responses/cs/HassLightSet.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+language: cs
+responses:
+  intents:
+    HassLightSet:
+      brightness: "Jas {{ slots.name }} nastaven na {{ slots.brightness }}"
+      brightness_area: "Jas {{ slots.area }} nastaven na {{ slots.brightness }}"
+      color: "Barva {{ slots.name }} nastavena na {{ slots.color }}"
+#      color_area: "Barva {{ slots.area }} nastavena na {{ slots.color }}"
```
Comment (halecivo) [inline responses/cs/HassLightSet.yaml:8]: can we already add
Barva světel v ...

File: sentences/cs/cover_HassCloseCover.yaml [hunk 2]
```diff
@@ -1,13 +1,15 @@
-language: "cs"
+language: cs
```
Comment (halecivo) [inline sentences/cs/cover_HassCloseCover.yaml:1]: this is being changed from cs to "cs" and back already multiple times, is there some final guideline which fotmat we should keep?

File: responses/cs/HassClimateGetTemperature.yaml [hunk 3]
```diff
@@ -2,4 +2,4 @@ language: cs
 responses:
   intents:
     HassClimateGetTemperature:
-      default: '{{ state.state }} stupňů'
+      default: '{{ state.state }} {{ ''stupeň'' if state.state == 1 else ''stupně'' if state.state in [2,3,4] else ''stupňů'' }}'
```
Comment (halecivo) [inline responses/cs/HassClimateGetTemperature.yaml:5]: Official language would say also "21 stupeň", but for most of people "21 stupňů" sounds more natural.

For negative values we keep "stupňů" everywhere?

File: responses/cs/HassLightSet.yaml (hunk 1 above)
Comment (schizza) [reply responses/cs/HassLightSet.yaml:8]: Yes, there will be "Barva v <area>" in next commit. I have prepared it. But then migration come in place.

Good point with negative temp values. 
And I think, nobody will ever use "21 supeň" as it is gramaticly correct, but mosty used for ordinal numbers - 21. stupeň

**Comment by halecivo** [comment]
The PR has quite lot of conflicts with recently merged PR for responses. I checked few of them and they seem to be great, but might be good to check again once you resolve them. 👍

**Comment by schizza** [comment]
Yes I am awared of conflicts, since there was a major update @synesthesiam posted PR with English sentences that is covered in this PR. 

It is closed now and I will post new one to reflect merge update.

**Pull request closed without merging**